sematic/utils/git.py

```diff
@@ -24,32 +24,32 @@ def get_git_info(object_: Any) -> Optional["Repo"]:  # type: ignore # noqa: F821
         logger.warn("Could not get git information", exc_info=e)
         return None
 
+    # try to search the path ancestors of the given object's source code for a git repo
```

Comment:
I realized using the bazel workspace changes the way this func should be documented: it's now trying to get the git repo for the current submission, either from the object or elsewhere

Reply:
Done. Added cwd support as well.